src/Microsoft.AspNetCore.Server.Kestrel/KestrelServer.cs

```diff
@@ -100,6 +100,10 @@ public void Start<TContext>(IHttpApplication<TContext> application)
                         "ThreadCount must be positive.");
                 }
 
+                if (!Constants.ECONNRESET.HasValue)
+                {
+                    _logger.LogWarning("Unable to determine ECONNRESET value on this platform.");
+                }
```

Comment:
lol

when I see two `if` blocks like that I sometimes mentally insert a non-existing `else` before the second `if` when reading the code.

But this is not really critical so you can leave it as is.